rules.md

```diff
@@ -12,44 +12,46 @@ The game consists of a series of cards and tokens.
 
 ## Set-up
 
-- Shuffle the **Role** cards together to create the **Roles Deck**.
-- Each **Player** chooses / is assigned a **Role** card from the **Roles Deck**.
```

Comment:
We should add a section at the end of the rules that has some ideas for varying the game, of which one of the ideas is to randomly deal a role card rather than let the players choose. I know it sounds obvious, but should help to keep the replay value up.

Reply:
We could add a `House rules` section where we say people can house rule it as they like (some will anyway), but also invite them to submit a play test report that also documents the house rules they played with. (Open game development!)

Reply:
This is a good idea but unrelated to this PR. I've created an issue to keep track of the task here #8